File: HW4/Drinkables.java
package HomeWork.HW4;

import java.math.BigDecimal;

public class Drinkables extends Product {
    private final Double volume;

    public Drinkables(String name, BigDecimal price, Double volume) {
        super(name, price);
        this.volume = volume;
    }

    public Double getVolume() {
        return volume;
    }

    @Override
    public String toString() {
        return super.toString() + "объем = " + volume + " ";
    }

}

File: HW7/model/products/Drinkables.py
from HW7.model.products.Product import Product


class Drinkables(Product):
    def __init__(self, name, price, volume):
        super().__init__(name, price)
        self._volume = volume

    @property
    def volume(self):
        return self._volume

    @volume.setter
    def volume(self, new_volume):
        self._volume = new_volume

    def __str__(self) -> str:
        return f"{super().__str__()} руб., объем: {self.volume} литра"